tests/unit/models/transactions/test_delete_oracle.py

```diff
@@ -0,0 +1,14 @@
+from unittest import TestCase
```

Comment:
This file should be renamed to `test_oracle_delete.py` to match transaction name. Same for `test_oracle_set.py`

Reply:
thanks. fixed in db3d27f